A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg

```
@@ -15,11 +15,11 @@ lightdef.8 = Type:6#Index:3#LocalPosition:0,0,0#LocalRotation:10,0,55#EffectFile
 lightdef.9 = Type:5#Index:1#LocalPosition:0,0,0#LocalRotation:5,0,0#EffectFile:LIGHT_A32NX_TakeOff#Node:LIGHT_ASOBO_TakeOff#PotentiometerIndex:1#EmMesh:LIGHT_ASOBO_TakeOff
 lightdef.10 = Type:5#Index:2#LocalPosition:-10,-7.6,-0.7#LocalRotation:5,10,10#EffectFile:LIGHT_A32NX_LandingLarge#Node:#PotentiometerIndex:1#EmMesh:LIGHT_ASOBO_LandingLeft
 lightdef.11 = Type:5#Index:3#LocalPosition:-10,7.6,-0.7#LocalRotation:5,-10,-10#EffectFile:LIGHT_A32NX_LandingLarge#PotentiometerIndex:1#EmMesh:LIGHT_ASOBO_LandingRight
-lightdef.12 = Type:2#Index:0#LocalPosition:-79.15,-0.03,7.77#LocalRotation:0,0,180#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeTail#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeTail
-lightdef.13 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,90#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftOuter
-lightdef.14 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,20#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftInner
-lightdef.15 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-90#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightOuter
-lightdef.16 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-20#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightInner
+lightdef.12 = Type:2#Index:0#LocalPosition:-79.15,-0.03,7.77#LocalRotation:0,0,180#EffectFile:LIGHT_A32NX_StrobeTail#Node:LIGHT_ASOBO_StrobeTail#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeTail
+lightdef.13 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftOuter#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingLeftOuter
+lightdef.14 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftInner#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingLeftInner
+lightdef.15 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightOuter#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingRightOuter
+lightdef.16 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightInner#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingRightInner
```

```suggestion
lightdef.13 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftOuter
lightdef.14 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftInner
lightdef.15 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightOuter
lightdef.16 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightInner
```

The strobe potentionemter index was reverted back to 24 in #861